build/Linux+BSD/portable/copy-libs

```diff
@@ -116,8 +239,6 @@ building_on_CentOS_6() {
   copyLib libQt5Concurrent.so.5
   copyLib libQt5DBus.so.5
   copyLib libQt5Designer.so.5
-  copyLib libQt5Location.so.5
-  copyLib libQt5QuickParticles.so.5
```

Comment:
fyi, all I have in my arm AppImage under qt5/qml is QtQml, QtQuick, QtQuick.2.  I'm actually not feeling comfortable removing anything more at the moment, becuase I'm afraid I'm going to break something.